Makefile

```diff
@@ -106,6 +107,9 @@ reindex:
 flake8:
 	flake8 src/
 
+flake8-dev:
+	(git diff --cached --name-only; git diff --name-only) | sort -u | grep '.py$$' | xargs flake8
```

Comment:
@kewisch the `$$` causes the list of changes files to be blank. Shouldn't that be `$`?